docs/contributing/platforms/versioning.mdx

```diff
@@ -0,0 +1,70 @@
+---
+title: Versioning
+sidebar_order: 2
+---
+
+
+Versioning allows for the maintenance of multiple versions of the same documentation, which is crucial for supporting different SDK releases. This guide explains how to create versioned documentation in the docs platform.
+
+## Creating a Versioned Page
+
+To create a versioned page:
+
+1. Ensure the original page already exists.
+2. Create a new file with the same name as the original, but append `__v{VERSION}` to the filename (before the extension).
+
+For example, to create version 2.2.0 of `index.mdx`:
+
+```
+index.mdx (original)
+index__v2.0.0.mdx (version 2.0.0)
+```
+
+## Versioning Platform Includes
+
+Pages in the `platform-includes` folder can be versioned using the same method. However, the root page referencing these includes must also be versioned.
+
+## File Structure Example
+
+```
+docs/
+├── getting-started/
+│   ├── index.mdx
+│   └── index__v2.mdx
+└── platform-includes/
+    └── configuration/
+        ├── example.mdx
+        └── example__v2.mdx
+```
+
+## Version Selection and User Experience
+
+When multiple versions of a page exist:
+
+1. A dropdown menu appears below the platform selector, displaying all available versions for the current file.
+
+2. When a user selects a version:
+
+- They are redirected to the chosen version of the page.
+- A preference is stored in localStorage, specific to the current guide or platform.
+
+3. An alert is displayed when viewing "non-latest" pages to inform users they're not on the most recent version.
+
+4. If a user has a stored version preference:
+
+- The app will perform a client-side redirect to this version whenever it's available for other pages.
+
+This approach ensures users can easily navigate between versions and keep their version preference across the documentation.
+
+## Best Practices
+
+1. Only add versioning to pages when necessary, to avoid content duplication and save build times.
+2. Ensure all related content (including `platform-includes`) is versioned consistently.
```

Comment:
This is my only concern that we could easily become inconsistent. Should we add a convention in this guide?

I'd suggest something like 
- `__v<major>.x` for versioning a page for a specific major version range (like in the express example)
- `__v<major>.<minor>.<patch>`versioning a page for a specific version (like in the NextJS example)

We could possibly enfore this with a regex check I guess but probably not necessary for the moment 🤔

Reply:
@Lms24 Yep that is probably something we should consider. Like crashing the build when the version does not follow this convention?

Reply:
sounds great, thanks!